lib/array_database.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class ArrayDatabase
+  def initialize
+    @all = []
+  end
+
+  attr_reader :all
```

Comment:
When attributes of a class are public like this, they usually go at the beginning of the class (before the constructor) :+1: